mavros/0e2ea0c/0e2ea0c.bug

```diff
@@ -45,9 +45,15 @@ fix:
   fix-in:
     - mavros/src/plugins/ftp.cpp
   languages: C++
-  time: 2015-07-29 (00:00)
+  time: 2015-07-29T13:27:14+03:00
```

Comment:
Where did you find this timestamp? `get_issue_creation_date.py` returns everything in `Z`.

Reply:
`git log`.